avocado/core/extension_manager.py

```diff
@@ -32,6 +33,15 @@
 LOG_UI = logging.getLogger("avocado.app")
 
 
+class PluginPriotiy(enum.IntEnum):
```

Comment:
any chance this can be changed to `PluginPriority`?

Reply:
Yes, you are right. This is a typo.